ephios/plugins/basesignup/signup/section_based.py

```diff
@@ -234,37 +236,52 @@ def get_configuration_fields(self):
             },
         }
 
+    def _get_signup_stats_per_section(self, participations=None):
+        from ephios.core.signup.methods import SignupStats
+
+        if participations is None:
+            participations = list(self.shift.participations.all())
+        confirmed_counter = Counter()
+        requested_counter = Counter()
+        for p in participations:
+            if p.state == AbstractParticipation.States.CONFIRMED:
+                c = confirmed_counter
+            elif p.state == AbstractParticipation.States.REQUESTED:
+                c = requested_counter
+            else:
+                continue
+            c[p.data.get("dispatched_section_uuid", NO_SECTION_UUID)] += 1
```

Comment:
requested participations are put into the section "" rather than "other". This makes the whole counting process go wrong and the requested_count in the resulting signup stats is always 0

Reply:
Very good catch, though I think the defect was a littler different/not only this. See a8f7fba for the refactor.

Reply:
well but as the stats have been summed up before i think just fixing this line should have solved it?